Tests/SwiftBuildTests/ProjectModel/TargetDependencyTests.swift

```diff
@@ -16,9 +16,7 @@ import Testing
 
 @Suite
 fileprivate struct TargetDependencyTests {
-    @Test func basicEncoding() throws {
```

Comment:
FYI there's several tests called basicEncoding, the one I saw hang was SandboxingOverrideTests.basicEncoding()